examples/Makefile.am

```diff
@@ -25,27 +25,27 @@ noinst_PROGRAMS = mapnotify simplelog tcpclient tcpserver ipcclient ipcserver
 
 mapnotify_SOURCES = mapnotify.c $(top_builddir)/include/qb/qbmap.h
 mapnotify_CPPFLAGS = -I$(top_builddir)/include -I$(top_srcdir)/include
-mapnotify_LDADD = $(top_builddir)/lib/libqb.la
+mapnotify_LDADD = $(top_builddir)/lib/libqb.la $(SYSTEMD_LIBS)
```

Comment:
I don't think this adding $(SYSTEMD_LIBS) everywhere is good idea (and if it really is, then ` libqb.pc` must be changed).

What about changing lib/Makefile.am` and add $(SYSTEMD_CFLAGS) and $(SYSTEMD_LIBS) to libqb_la_CFLAGS and libqb_la_LIBADD? Then libqb.la gets dependency on systemd (when needed) and apps using libqb will need no change? (please correct me if I'm wrong, I'm not libtool magic master).

Reply:
Possibly. I'm struggling with auto-stuff  too, It will definitely need to go in the pkgfile somehow